src/MongoDB.Analyzer/Core/Utilities/AnalysisCodeGeneratorUtilities.cs

```diff
@@ -56,5 +59,7 @@ analysisType switch
             AnalysisType.Poco => JsonSyntaxElements.Poco.JsonGeneratorFullName,
             _ => throw new ArgumentOutOfRangeException(nameof(analysisType), analysisType, "Unsupported analysis type")
         };
+
+    private static bool IsDriverVersion_2_28_Or_Greater(string driverVersion) => VersionRange.Parse(Driver_V2_28_OrGreater).Satisfies(NuGetVersion.Parse(driverVersion));
```

Comment:
Consider having `readonly Version s_driverVersion_2_28`, and comparing 
```
referencesContainer.Version >=
 s_driverVersion_2_28
```
We don't actually need range here.

Reply:
Done